src/BinDeps.jl

```diff
@@ -47,8 +47,8 @@ module BinDeps
     function download_cmd(url::AbstractString, filename::AbstractString)
         global downloadcmd
         if downloadcmd === nothing
-            for download_engine in @windows? ("C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell",
-                    :powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
+            for download_engine in (@static is_windows()? ("C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell",
```

Comment:
the deprecation message is accurate because os_only was a macro, but the majority of uses don't actually need to be a macro.